canonical-websites/www.ubuntu.com
# Update /download/server/s390x - 18.04

## Done

Rename and redirect linuxone page and some copy updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/s390x>
- Check page matches [the copy doc](https://docs.google.com/document/d/1d85IfqqxZYYZ0orj5lBhqLWLGpLYWRLdvs75lYsplGs/edit)
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/linuxone>
- Make sure page redirects to <http://0.0.0.0:8001/download/server/s390x>


## Issue / Card
 Fixes #2870
